engine/src/main/java/io/zeebe/engine/nwe/behavior/BpmnStateTransitionBehavior.java

```diff
@@ -255,14 +254,17 @@ public void onElementTerminated(
     containerProcessor.onChildTerminated(containerScope, containerContext, childContext);
   }
 
-  private ExecutableCallActivity getParentWorkflowScope(
+  private ExecutableFlowElement getParentWorkflowScope(
       final BpmnElementContext callActivityContext, final BpmnElementContext childContext) {
     final var workflowKey = callActivityContext.getWorkflowKey();
     final var elementId = callActivityContext.getElementId();
+    final var elementType = callActivityContext.getBpmnElementType();
```

Comment:
So I would change this to:
```suggestion
    final var elementType = BpmnElementType.CALL_ACTIVITY;
```

Reply:
:+1: